build.gradle

```diff
@@ -12,11 +12,18 @@ repositories {
     maven {
         url "https://dl.bintray.com/embulk-base-restclient/maven"
     }
+    maven {
```

Comment:
Gem should public as a public gem, not private under treasuredata host

Reply:
updated